File: ago-dic-2023/Angel Carrillo Berlanga/Practica1/practica1.py
entrada = input("Introduce dos enteros separados por un espacio: ")

numeros = entrada.split()

if len(numeros) != 2:
    print("Por favor, ingresa dos enteros separados por un espacio.")
else:
    try:
        numero1 = int(numeros[0])
        numero2 = int(numeros[1])
        
        suma = numero1 + numero2
        
        print("Resultado:", suma)
    except ValueError:
        print("Error: Asegúrate de ingresar dos enteros válidos.")
2

File: ago-dic-2023/practicas/oliver misael guzman miramontes/practica1/practica1.java
import java.util.Scanner;
public class practica1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Ingrese dos enteros separados por un espacio:");
        
       
        int numero1 = sc.nextInt();
        int numero2 = sc.nextInt();
        
       
        int suma = numero1 + numero2;
        
       
        System.out.println("La suma de " + numero1 + " y " + numero2 + " es: " + suma);
        
       
      
    }
}
